### PR TITLE
Ruchir day in

### PR DESCRIPTION
**Pull request by ruchirt**

### CODE REVIEW

File: query_builder/app/queryfilters/__init__.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+                    'multivalue': MultiValueQueryFilter
+                    }
+
+
+def build_filters():
+    filters = {}
+    for filter_def in settings.COMPANIES_FILTERS:
+        filter_name = filter_def['filter']
+        filter_type = filter_def['type']
+        filter_options = None
```
Comment (pmajmudar) [inline query_builder/app/queryfilters/__init__.py]: if you set filter_options to an empty dict:
`filter_options = {}`
You can always call the filter with `(filter_name, **filter_options)`

Then you can remove the additional else block - for readability.

File: query_builder/app/queryfilters/numeric_range_query_filter.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+        if value_obj:
+            exp = re.compile("^(\-?[0-9]+)?\-(\-?[0-9]+)?$")
+            m = re.search(exp, value_obj)
+            if not m:
+                raise exceptions.ParameterValueError(key=self.id, value=value_obj)
+
+            lbound, ubound = None, None
+            # Parse lower bound
+            if m.group(1):
```
Comment (pmajmudar) [inline query_builder/app/queryfilters/numeric_range_query_filter.py]: This code block is repeated for m.group(1) and m.group(2) - we could probably factor that out to clean up the code.

File: query_builder/tests/end_to_end/test_booleans.py [hunk 3]
```diff
@@ -10,8 +14,53 @@ def test_boolean_true():
...
     url = "/v1/company_query_builder?ecommerce=false"
     response = get_es_query(url)
-    assert response == full_es_query(None)
\ No newline at end of file
+    assert response == full_es_query(None)
+
+
+def test_boolean_query_filter():
+    f = BooleanQueryFilter("abc", include_if_false=False)
+    assert True == f.parse(["1"])
```
Comment (pmajmudar) [inline query_builder/tests/end_to_end/test_booleans.py]: Ideally we would use pytest.mark.parametrize as a decorator to move the test cases out of the logic of the test.

If you're not familiar with it - we can pair on it.

File: query_builder/app/queryfilters/__init__.py (hunk 1 above)
Comment (ruchirt) [reply query_builder/app/queryfilters/__init__.py]: fixed

File: query_builder/app/queryfilters/numeric_range_query_filter.py (hunk 2 above)
Comment (ruchirt) [reply query_builder/app/queryfilters/numeric_range_query_filter.py]: fixed

**Pull request closed without merging**